lib/Installation/EncryptedVolume/EncryptedVolumeController.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Controller for YaST Encrypted Volume module.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::EncryptedVolume::EncryptedVolumeController;
```

Comment:
just a suggestion here: s/EncryptedVolumeController/EncryptedVolumeActivationController/
The activation part is really the key of the logic of this screen, at least as minimum, let's add it to the Summary.

Reply:
Done